tests/unit/backend/corpora/common/entities/test_project.py

```diff
@@ -0,0 +1,38 @@
+import os
+import sys
+import unittest
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..", "..", "..", "..", ".."))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from backend.corpora.common.entities.project import Project
+from backend.corpora.common.utils.exceptions import CorporaException
+
+
+class TestProject(unittest.TestCase):
+    def setUp(self):
+        self.uuid = "f753497f-f8a5-48b5-b344-de9ad5a4354d"
+        self.status = "LIVE"
+
+    @unittest.skipIf(True, "Only runnable on local dev env via bastion tunnel. Comment this line to run.")
```

Comment:
You can move this decorator to `class TestProject` if it's used on all of the tests.

Reply:
Deprecated